rocky/rocky/settings.py

```diff
@@ -78,7 +78,7 @@
     "loggers": {
         "root": {
             "handlers": ["console"],
-            "level": "INFO",
+            "level": env("LOG_LEVEL", default="WARNING").upper(),
```

Comment:
This increases the log level default from info to warning, for the entire Rocky app.